spec/integ/matrix-client-room-timeline.spec.js

```diff
@@ -252,11 +252,14 @@ describe("MatrixClient room timelines", function() {
                 client.scrollback(room).done(function() {
                     expect(room.timeline.length).toEqual(1);
                     expect(room.oldState.paginationToken).toBe(null);
-                    done();
+
+                    // still have a sync to flush
+                    httpBackend.flush("/sync", 1).then(() => {
```

Comment:
See the commit comment (https://github.com/matrix-org/matrix-js-sdk/pull/499/commits/adca75b7d89d91d5d8a019e0f873a023a69d2ddb): the `afterEach` checks it's been flushed.